test/integration/framework/src/siftool/sifchain.py

```diff
@@ -172,7 +192,7 @@ def sifnoded_start(self, tcp_url=None, minimum_gas_prices=None, log_format_json=
         return self.cmd.spawn_asynchronous_process(sifnoded_exec_args, log_file=log_file)
 
     def build_start_cmd(self, tcp_url=None, minimum_gas_prices=None, log_format_json=False):
-        args = [self.binary, "start"] + \
+        args = [self.binary, "start", "--trace"] + \
```

Comment:
Do we still want/need this?

Reply:
Yes, start with trace can help to get the stack if something wrong. It not change the trace level, just print out the stack if panic.